languages/ru-RU.js

```diff
@@ -0,0 +1,196 @@
+			INHIBITOR_MISSING_BOT_PERMS: (missing) => `Недостаточно прав для выполнения команды, отсутствуют следующие разрешения: **${missing}**`,
+			INHIBITOR_NSFW: 'Вы можете использовать NSFW команды только в NSFW каналах.',
+			INHIBITOR_PERMISSIONS: 'У вас недостаточно прав для использования этой команды.',
+			INHIBITOR_REQUIRED_SETTINGS: (settings) => `Команда не может быть выполнена, так как на сервере отсутствуют следующие настройки: **${settings.join(', ')}**.`,
+			INHIBITOR_RUNIN: (types) => `Эта команда может быть выполнена только в определённых типах каналов: ${types}.`,
+			INHIBITOR_RUNIN_NONE: (name) => `Команда ${name} не настроена для выполнения ни в одном канале.`,
+			COMMAND_BLACKLIST_DESCRIPTION: 'Добавляет/удаляет пользователей и серверы в чёрный список бота.',
+			COMMAND_BLACKLIST_SUCCESS: (usersAdded, usersRemoved, guildsAdded, guildsRemoved) => [
+				usersAdded.length ? `**Пользователи добавлены**\n${util.codeBlock('', usersAdded.join(', '))}` : '',
+				usersRemoved.length ? `**Пользователи удалены**\n${util.codeBlock('', usersRemoved.join(', '))}` : '',
+				guildsAdded.length ? `**Серверы добавлены**\n${util.codeBlock('', guildsAdded.join(', '))}` : '',
+				guildsRemoved.length ? `**Серверы удалены**\n${util.codeBlock('', guildsRemoved.join(', '))}` : ''
+			].filter(val => val !== '').join('\n'),
+			COMMAND_EVAL_DESCRIPTION: 'Выполняет произвольный JavaScript. Зарезервировано для владельца бота.',
+			COMMAND_EVAL_EXTENDEDHELP: [
+				'Команда eval выполняет код внутри себя, любая ошибка будет обработана.',
+				'Так же вы можете использовать флаги --silent, --depth=число и --async для изменения поведения команды.',
+				'Флаг --silent отключает вывод результата.',
+				'Флаг --depth принимает число, например --depth=2, которое задаёт глубину результата для util.inspect.',
+				'Флаг --async оборачивает код в асинхронную функцию, благодаря чему вы сможете использовать await, но вам так же придётся использовать return чтобы вернуть результат.',
+				'Флаг --showHidden включает параметр showHidden в util.inspect.',
+				'Если результат слишком большой, он будет отправлен в виде файла или выведен в консоль, если бот не имеет разрешения на отправку файлов.'
+			].join('\n'),
+			COMMAND_EVAL_ERROR: (time, output, type) => `**Ошибка**:${output}\n**Тип**:${type}\n${time}`,
+			COMMAND_EVAL_OUTPUT: (time, output, type) => `**Результат**:${output}\n**Тип**:${type}\n${time}`,
+			COMMAND_EVAL_SENDFILE: (time, type) => `Результат оказался слишком длинным, он будет отправлен в виде файла.\n**Тип**:${type}\n${time}`,
+			COMMAND_EVAL_SENDCONSOLE: (time, type) => `Результат оказался слишком длинным, он будет выведен в консоль.\n**Тип**:${type}\n${time}`,
+			COMMAND_UNLOAD: (type, name) => `✅ Выгружено ${type}: ${name}`,
+			COMMAND_UNLOAD_DESCRIPTION: 'Выгружает элемент klasa.',
+			COMMAND_UNLOAD_WARN: 'Скорее всего вам не стоит выгружать это, так как вы не сможете использовать какую-либо команду чтобы загрузить это снова.',
```

Comment:
A comma before "чтобы" is needed.
```suggestion
			COMMAND_UNLOAD_WARN: 'Скорее всего вам не стоит выгружать это, так как вы не сможете использовать какую-либо команду, чтобы загрузить это снова.',
```